core/core.go

```diff
@@ -101,6 +105,12 @@ type NetworkDriver interface {
 	DeleteMaster(node ServiceInfo) error
 	AddBgp(id string) error
 	DeleteBgp(id string) error
+	// Add a service spec to proxy
+	AddSvcSpec(svcName string, spec *ofnet.ServiceSpec) error
```

Comment:
it should be ok.
So far we have tried to keep ofnet as an independent library thats not dependent on netplugin.
It'll be nice to keep it that way..